diagnostic.md

```diff
@@ -45,6 +55,7 @@ weather === "sunny"
 
 What do these expressions evaluate to?  Explain your answers.
 
+true ; because weather holds the value "sunny", and truthy and truthy is a true
```

Comment:
Okay, we also wanted you to list what the first two lines evaluated to but I'll assume you knew!